app/mailers/noisy_workflow.rb

```diff
@@ -15,6 +15,7 @@ def make_noise(action)
 
   def skip_review(action)
     @edition = action.edition
+    @edition_url = edition_url(@edition.id, host: Plek.find("publisher"))
```

Comment:
The default host not being Publisher is weird...

Anyway, you'll probably want to pass `external: true` to Plek, as this'll get you a "external" URL. In Carrenza, the external URLs are the same as the internally (app to app) ones, but in AWS they're different:

```
irb(main):007:0> Plek.find('publisher')
=> "https://publisher.integration.govuk-internal.digital"
irb(main):008:0> Plek.find('publisher', external: true)
=> "https://publisher.integration.publishing.service.gov.uk"
```